README.md

```diff
@@ -4,30 +4,34 @@ Order files can reduce app startup time by co-locating symbols that are accessed
 file by launching the app in an XCUITest. Read all about how order files work in [our blog post](https://www.emergetools.com/blog/posts/FasterAppStartupOrderFiles).
 
 Setting up your app for order files requires 2 steps:
-1 - Use this library to generate an order file as part of your XCUITest. The library instruments app launch in the UI test and uses the results to generate an optimized order file
-2 - Re-build the app with the order file. Once the order file is generated you build the app again, this time passing the order file as an option to the linker.
 
-## Installation
+1. Generate an order file as part of an XCUITest. FaultOrdering instruments app launch in the UI test and uses the results to generate an optimized order file.
+2. Once the order file is generated, build the app again, this time passing the order file as an option to the linker.
```

Comment:
Made these a markdown ordered list

Reply:
Yep, they're still above the package installation header. After incorporating your feedback they're actually before `## Setup` as well 👍🏻